src/mudlet.cpp

```diff
@@ -3778,6 +3778,8 @@ void mudlet::slot_connection_dlg_finished(const QString& profile, bool connect)
     pHost->getScriptUnit()->compileAll();
     pHost->mIsProfileLoadingSequence = false;
 
+    pHost->updateAnsi16ColorsInTable();
```

Comment:
Isn't this duplicating what the Host constructor already did?

Reply:
The one in the constructor populates it with the ones that are in the constructor's initialiser list - however a loaded profile can override those with ones the user has set for the profile. I suppose it might be the case that the first of those two uses is unneeded - but what happens for a newly created profile without a previous save file to load...